source/examples/xgboost-dask-databricks/notebook.ipynb

```diff
@@ -0,0 +1,2163 @@
+{
```

Comment:
<p>So far we've used Dask to convert from CSV to Parquet, then we are using Spark to Convert Parquert to Delta Lake, then we are ultimately doing to read it back in with Dask.</p><p> </p><p>Maybe we should just cut things down and use dask-deltatable to convert directly from CSV to Delta Lake and skip out Spark altogether?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>